src/components/application_manager/src/commands/mobile/add_sub_menu_request.cc

```diff
@@ -60,7 +60,7 @@ void AddSubMenuRequest::Run() {
   const int32_t menu_id =
       (*message_)[strings::msg_params][strings::menu_id].asInt();
   if (app->FindSubMenu(menu_id)) {
-    SDL_ERROR("Menu with id " << menu_id << " is not found.");
+    SDL_ERROR("Menu with id " << menu_id << " is already exists.");
```

Comment:
@LevchenkoS This was not changed since my previous comment although you claim it is fixed. I meant that " is already exists." is not correct. Better suggestions are "already exists." or "is already existing."